templates/data/mysql.html

```diff
@@ -1,477 +1,533 @@
-            alt="",
-            width="595",
-            height="384",
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-logo-section__logo"}
-            ) | safe
-          }}
-        </div>
-        <div class="p-logo-section__item">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/3045191f-maas-logo.png",
-            alt="",
-            width="247",
-            height="384",
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-logo-section__logo"}
-            ) | safe
-          }}
-        </div>
+      <div class="col-9 col-medium-4">
+        <div class="p-section--shallow">
+          <h1>MySQL for the enterprise</h1>
+          <p>
+            Secure and automate the deployment, maintenance and upgrades of your MySQL databases across private and public clouds.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="data-relational-dbs-modal">Get in touch<span class="u-off-screen">about MySQL for the enterprise</span></a>
+        <a class="p-button"
+           href="https://drive.google.com/file/d/1vQamR5wOA9E-FTIJG2c9xCWLpB_1QcA3/view?usp=sharing"><span class="u-off-screen">Download the Canonical's guide to MySQL</span>Get started</a>
       </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <hr class="p-rule is-fixed-width">
-  <div class="row--50-50">
-    <div class="col">
-      <h2>A hyper-automated MySQL made for high availability</h2>
-    </div>
-  
-    <div class="col">
-      <p>Thanks to our software operator for MySQL, your deployments will be:</p>
-      <hr class="p-rule is-muted">
-      <div class="p-section--shallow">
-        <ul class="p-list--divided u-no-margin--bottom">
-          <li class="p-list__item is-ticked"><strong>Highly available</strong> with built-in replication and automatic failover.</li>
-          <li class="p-list__item is-ticked"><strong>Reliable</strong> with expert-crafted and self healing state-machine automation.</li>
-          <li class="p-list__item is-ticked"><strong>Upgraded automatically</strong> during your maintenance windows and with minimum downtime.</li>
-          <li class="p-list__item is-ticked"><strong>Disaster recovery ready</strong> with built-in backup and restore capabilities to ensure business continuity.</li>
-          <li class="p-list__item is-ticked"><strong>Hybrid and multi-cloud ready</strong> &dash; Our automation supports deployment within and across private and public clouds.</li>
-          <li class="p-list__item is-ticked"><strong>Shipped with a holistic observability and alerting solution</strong> based on Prometheus, Loki and Grafana and Loki to monitor all of your database fleets.</li>
+  <section class="p-section">
+    <div class="row--25-75">
+      <hr class="u-no-margin--bottom" />
```

Comment:
No need to remove margin here. You have the same pattern in the next section that does have a margin.